raco/language/myrialang.py

```diff
@@ -566,6 +570,9 @@ def __init__(self, input):
     def num_tuples(self):
         return self.input.num_tuples()
 
+    def partitioning(self):
```

Comment:
Fair enough, the producer is certainly the only one with the partitioning information.